doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -180,13 +180,16 @@ Provisioning registration data onto device
 The Fast Pair standard requires provisioning the device with Model ID and Anti-Spoofing Private Key obtained during device model registration.
 In the |NCS|, the provisioning data is generated as a hexadecimal file using the :ref:`bt_fast_pair_provision_script`.
 
-When building the Fast Pair in the |NCS|, the build system automatically calls the Fast Pair provision script and includes the resulting hexadecimal file in the firmware (the :file:`merged.hex` file).
+When building the Fast Pair in the |NCS|, the build system automatically calls the Fast Pair provision script and includes the resulting hexadecimal file in the final firmware hexadecimal file that can be flashed onto the device.
+For devices with the :ref:`partition_manager` support for which the :kconfig:option:`CONFIG_PM_SINGLE_IMAGE` Kconfig option is automatically enabled by the Fast Pair build system, the ``bt_fast_pair`` partition is automatically created by the build system.
```

Comment:
I would also describe the possibility of manually defining the partition with the `pm_static_..yaml` file (you could possibly add a reference to some section of the Partition Manager). At the same time, it would be good to point the user to the sample that demonstrates the explicit way of defining the partition (Locator Tag and specific target).

With the automatic definition, I would point the user to the Input Device sample.